src/core/utils/components/VirtualScroll.tsx

```diff
@@ -0,0 +1,232 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+
+const getScrollableParent = (element: HTMLElement | null): HTMLElement => {
```

Comment:
You really don't want to just have scrollable container ref as a prop? That way you could remove all these helper functions, also users would know and could set scrollable container exactly where they want, avoiding bugs when scrollable container is automatically detected in bad place.

Reply:
ah man.. I dont know.. Will you always have a reference to scrollable container?
I guess I can change it if you feel thats better and covers more use cases..

Reply:
They should have in most of the cases. They might not have if they are using some 3rd party container.

Reply:
I am struggling to make it work...